src/components/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+import classes from '../ExpandedNode/ExpandedNode.module.scss';
+import indexClasses from '../../styles/index.module.scss';
+import withConfirmationPopup from '../withPopup/withConfirmationPopup';
+import styles from './ActionButton.module.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes, faCheck } from '@fortawesome/free-solid-svg-icons';
```

Comment:
All of the font awesome icons are loaded in `utils/fontawesomeLibrary.js`

Reply:
I disagree with the approach because it allows the app to silently fail if they haven't been imported, which is also hard to know without using the explicit import.  Also it looks like the explicit inclusion might allow for a smaller bundle size.

Regardless, I changed it to use the library import until we can change everything over.